bootstrap/viaproxy/src/main/java/org/geysermc/geyser/platform/viaproxy/GeyserViaProxyPlugin.java

```diff
@@ -75,6 +77,7 @@ public void onEnable() {
         this.onGeyserInitialize();
 
         ViaProxy.EVENT_MANAGER.register(this);
+        GeyserImpl.getInstance().eventBus().register(this, new GeyserServerTransferListener());
```

Comment:
Maybe move this to where `GeyserImpl.load()` is called (in onInitialize) - it'll look a bit cleaner as you're also assigning the geyser variable there